src/nft/components/explore/CarouselCard.tsx

```diff
@@ -209,12 +194,13 @@ export const CarouselCard = ({ collection, onClick }: CarouselCardProps) => {
               return null
             }
             return (
-              <MarketplaceRow
-                key={`CarouselCard-key-${collection.address}-${marketplace.marketplace}`}
-                marketplace={MARKETS_ENUM_TO_NAME[market]}
-                listings={marketplace.count.toString()}
-                floorInEth={marketplace.floorPrice.toString()}
-              />
+              <>
+                <TableElement>{MARKETS_ENUM_TO_NAME[market]}</TableElement>
+                <TableElement>
+                  {formatNumberOrString(marketplace.floorPrice, NumberType.NFTTokenFloorPrice)} ETH
```

Comment:
Is there a regression in not handling the null case for floor price or listings count?